<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>puzzle15</class>
 <widget class="QWidget" name="puzzle15">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>puzzle15</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_one">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_two">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_three">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_four">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eight">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_five">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_six">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_seven">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_twelve">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ten">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_thirteen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fourteen">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin:1px;</string>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fifteen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eleven">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(179, 0, 0); color: rgb(253, 225, 11); font-size: 18px; border-radius: 15px; margin: 1px;</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLabel" name="win_label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-8</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgba(71, 170, 10,0.8); color: white; font-size: 32px; text-align: center; padding:40%;</string>
   </property>
   <property name="text">
    <string>YOU WIN!!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
